addons/web/static/tests/helpers/mock_server.js

```diff
@@ -528,51 +528,52 @@ var MockServer = Class.extend({
         return modelFields;
     },
     /**
-     * Simulate a call to the search_panel_local_counters method.
-     * A model is added to the params to perform the adequate read_group.
+     * Simulate a call to the server '_search_panel_domain_image' method.
```

Comment:
To check after modification of models.py (need to do something concerning limit)